utils/getlastpositions.php

```diff
@@ -0,0 +1,77 @@
+<?php
```

Comment:
I think there is no need to create new script `utils/getlastpositions.php`. I suggest to reuse `getpositions.php` passing zero as user id (same as we do with track id) to get latest positions for all users.

Reply:
Ok. In that case, wouldn't it be better to add a third parameter (ex. "last")? That way you get the info you ask for, not the "default" one:
-if you want all positions from user 1 and track 1:  userid=1&track=1
-if you want last position from user 1: userid=1&last=1
-if you want las position from all users: last=1

Reply:
Sounds good